client/src/components/History/Content/Collection/JobStateSummary.js

```diff
@@ -6,8 +6,8 @@
  */
 import { STATES } from "../model/states";
 
-const NON_TERMINAL_STATES = [STATES.NEW, STATES.WAITING, STATES.QUEUED, STATES.RUNNING];
-const ERROR_STATES = [STATES.ERROR, STATES.DELETED];
+const NON_TERMINAL_STATES = ["new", "waiting", "queued", "running", "resubmitted", "upload"];
+const ERROR_STATES = ["error", "discarded", "deleted"];
```

Comment:
```suggestion
const ERROR_STATES = ["error", "failed", "deleted"];
```

I am not sure deleted is an error state per se, but if this is how it was before then that's fine.